microsoft-azure/latest/config/index.md

```diff
@@ -213,7 +213,7 @@ The Azure Connector provides a number of properties on installation and for cust
 | connector.az.deleted.containerName | Content is copied here once it has been deleted. This includes after content is removed from the recycle bin and after a 14 day grace period. In order for the content to be removed entirely it must be deleted manually by an Administrator. For more information on what a content store is, see [Manage content stores]({% link content-services/latest/admin/content-stores.md %}#manage-content-stores). |
 | connector.az.maxErrorRetries | The maximum number of attempts to retry reads or writes to the Blob container in case of failed transfers. The default is `3`. This configuration uses exponential retries. See [Exponential back-off retry policy](https://docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific#azure-storage){:target="_blank"} for more details. |
 | connector.az.tryTimeout | Indicates the maximum time (in seconds) allowed for any single try of an HTTP request. When transferring large amounts of data, the default tryTimeout will probably not be sufficient. To overcome this, we compute the upload try timeout using: {::nomarkdown}<pre>uploadTryTimeout = tryTimeout * anticipatedPayloadSizeInMb</pre>{:/} |
-| connector.az.apiEndpoint | Allows configuring a different Azure endpoint. The default value is `blob.core.windows.net`. The value of this property is concatenated with the `connector.az.account.name` property in order to build the final URL, for example, `https://<connector.az.account.name>.<connector.az.apiEndpoint>`.<br><br> Added in Azure Connector 5.0. |
+| connector.az.apiEndpoint | Allows configuring a different Azure endpoint. The default value is `blob.core.windows.net`. The value of this property is concatenated with the `connector.az.account.name` property in order to build the final URL, for example, `https://<connector.az.account.name>.<connector.az.apiEndpoint>`.<br><br> Added in Azure Connector 5.0.x. |

```

Comment:
Please revert this change - it's important to keep version when new properties were added.